src/components/BookForm.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+const BookForm = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // Logic to handle form submission
+    console.log('Form submitted:', title, author);
```

Comment:
### [OPTIONAL]
Kindly remove all the console.log statements before publishing the project because sometimes it leaks important data which may harm the owner. Maybe it seems not important now but in the future, it will be also it's considered as Best Practice to remove all the console.log statements as it's just for the testing purpose. 👍